IntegrationTestSuite/WebservicesIntegrationTests/WebservicesIntegrationTests.csproj

```diff
@@ -34,6 +34,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="DevExpress.Data.v19.2, Version=19.2.5.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a, processorArchitecture=MSIL" />

```

Comment:
Why are there DevExpress libraries in integration tests? Should not be the case.